src/models/events/RoomEvent.ts

```diff
@@ -17,6 +17,48 @@ export interface TypicalUnsigned {
     [prop: string]: any;
 }
 
+export interface RoomEventData<T extends (Object | unknown) = unknown> {
```

Comment:
The SDK isn't really set up to have these interfaces exist: events are either `any` (deliberately) or an instance of `MatrixEvent` - neither of which rely on the interfaces existing. Eventually the SDK will publish a breaking change that actually uses `MatrixEvent` instances everywhere, however until then it's expected that new code does the conversion. See `getEventContext` for an example.

Reply:
I'll update the code to use MatrixEvent for now then, to make that transition easier.